lxd/api_cluster.go

```diff
@@ -1044,6 +1045,122 @@ func clusterNodesGet(d *Daemon, r *http.Request) response.Response {
 	return response.SyncResponse(true, urls)
 }
 
+// swagger:operation POST /1.0/cluster/members cluster cluster_members_post
+//
+// Request a join token
+//
+// Request a join token to add a cluster member.
```

Comment:
Should be `Requests a join token to add a cluster member.` to match the pattern of the others.

The short description is correct with just `Request a join token` though as that's the action from the user's perspective whereas the longer one is the function's description.